test/minikube/minikube_base_restore_test.go

```diff
@@ -436,3 +435,91 @@ func TestKubernetesAutoRestore(t *testing.T) {
+	output, err = k8s.RunKubectlAndGetOutputE(t, kubectlOptions, "exec", smPod, "-c", "engine", "--", "bash", "-c",
+		"echo \"{\\\"id\\\": \\\"123abc\\\"}\" > /var/opt/nuodb/archive/nuodb/demo/backup.json")
+	require.NoError(t, err, output)
+
+	output, err = k8s.RunKubectlAndGetOutputE(t, kubectlOptions, "exec", smPod, "-c", "engine", "--", "bash", "-c",
+		"echo \"{\\\"id\\\": \\\"123abc\\\"}\" > /var/opt/nuodb/journal/nuodb/demo/backup.json")
+	require.NoError(t, err, output)
+
+	defer testlib.Teardown(testlib.TEARDOWN_SNAPSHOT)
+
+	achiveVolumeName := "archive-volume-" + smPod
+	archiveSnapshotName := "archive-snapshot"
+	testlib.SnapshotVolume(t, namespaceName, achiveVolumeName, archiveSnapshotName)
+
+	journalVolumeName := "journal-volume-" + smPod
+	journalSnapshotName := "journal-snapshot"
+	testlib.SnapshotVolume(t, namespaceName, journalVolumeName, journalSnapshotName)
+
+	helm.DeleteE(t, &helm.Options{KubectlOptions: k8s.NewKubectlOptions("", "", namespaceName)}, sourceDatabaseChartName, true)
+
+	restoredDb := "db-clone"
+	testlib.StartDatabase(t, namespaceName, admin0, &helm.Options{
+		SetValues: map[string]string{
+			"database.sm.resources.requests.cpu":                                 testlib.MINIMAL_VIABLE_ENGINE_CPU,
+			"database.sm.resources.requests.memory":                              testlib.MINIMAL_VIABLE_ENGINE_MEMORY,
+			"database.te.resources.requests.cpu":                                 testlib.MINIMAL_VIABLE_ENGINE_CPU,
+			"database.te.resources.requests.memory":                              testlib.MINIMAL_VIABLE_ENGINE_MEMORY,
+			"database.sm.hotCopy.journalPath.persistence.storageClass":           testlib.SNAPSHOTABLE_STORAGE_CLASS,
+			"database.persistence.storageClass":                                  testlib.SNAPSHOTABLE_STORAGE_CLASS,
+			"database.name":                                                      restoredDb,
+			"database.persistence.dataSourceRef.kind":                            "VolumeSnapshot",
+			"database.persistence.dataSourceRef.name":                            archiveSnapshotName,
+			"database.persistence.dataSourceRef.apiGroup":                        "snapshot.storage.k8s.io",
+			"database.sm.hotCopy.journalPath.persistence.dataSourceRef.kind":     "VolumeSnapshot",
+			"database.sm.hotCopy.journalPath.persistence.dataSourceRef.name":     journalSnapshotName,
+			"database.sm.hotCopy.journalPath.persistence.dataSourceRef.apiGroup": "snapshot.storage.k8s.io",
+			"database.autoImport.backup_id":                                      "123abc",
+			"database.sm.hotCopy.journalPath.enabled":                            "true",
+		},
+	})
+
+	output, err = k8s.RunKubectlAndGetOutputE(t, kubectlOptions, "exec", admin0, "-c", "admin", "-c", "admin", "--", "bash", "-c",
+		fmt.Sprintf("echo \"SELECT id FROM testtbl;\" | nuosql %s --user dba --password secret", restoredDb))
+
+	require.NoError(t, err, output)
+
+	require.True(t, strings.Contains(output, "123"))
```

Comment:
I'd like to see this test write more data to the database and then verify that it is present after a restart.
1. Write more data.
2. Delete the SM pod.
3. Wait for the database to become running again.
4. Check that the data written in 1 is present.

This verifies that we can actually diverge from the snapshot/dataSource and also exercises the "normal restart" code paths in `nuosm`.

Reply:
Sure thing.

Reply:
Is this implemented in a different test case?